Project-2/README.md

```diff
@@ -49,11 +118,11 @@ Note that we can drop the first universal quantifiers :)
 
 (c) ∀y(R(x, y) → ∃x(R(y, x) ∧ ∀y(R(x, y) → A(y)))) - For all y if there is some x thats Rs all instances of y then there is some x such that all instances of y Rs some x and for all y if x Rs y then y As
 
-Is this well-formed? I think not. For it to be well formed it should look closer to  ((∀y∃x(R(x, y) → R(y, x)) ∧ (R(x, y) → A(y)))
+Is this well-formed? I think not. For it to be well-fromed there should be another set of parenthesis to make it a conjunction additionally there is an unbound x
 
 (d) (∀y)(R(x, y) → A(y)) ∧ (∃y)(R(x, y) ∧ B(y)) - For all y if x Rs y then y As and some instances of y x Rs y and y Bs
 
-Is this well-formed? I think not. For it to be well-fromed there should be another set of parenthesis to make it a conjunction
+Is this well-formed? I think not. There is an unbound x!
```

Comment:
Q6. (b) ∃r-.C and ∀r-.C
“(b) Corresponds to some instance of C that are related to all instances
(b) Corresponds to only instances of C that are related to all instances”
-- The first refers to all individuals which have some instance C as a r-predecessor. The second refers to all individuals which have only instances of C as r-predecessors.

“(c) Corresponds to restrictions that r is related to no more than n instances
(c) Corresponds to restrictions that r is realted to no more than n instances that relates to C”
-- r is a relation but not an individual, so you cannot say “r is related to ….” 
The first one refers to all individuals which have no more than n r-fillers.
The second one refers to all individuals which are r-related to no more than n instances of C. Just say “n instances of C” but not “n instances that relates to C”.

(d) Corresponds to some instance of C that are related to all instances
(d) Corresponds to some instance of C that are related to the instance mapped to by the name a
(a, b): r means that b is a r-filler or r-successor of a; (a, b): r- means that b is a r-predecessor of a.
So the first one should mean all individuals which has some instance of C as r-predecessor. Note that in DL all quantified formulas include an implicit universal quantifier, so *each quantified formula in DL must pick out “all individuals satisfying some complex condition”*.

The second one refers to all individuals which have some r-predecessor which is just a.
Q7. Finite model property (FMP) can be used to design a decidable algorithm for the satisfiability of its concepts.

Q8. TMP is crucial to tableau-based algorithms for the consistency check of knowledge base and applied ontology. Most of applied ontologies, especially those BFO-based ones, have TMP, allowing that their consistency can be checked by tableau reasonings. 

Q10. I think you need to add a new picture to show “b is part of f” in (c)

Reply:
regarding 6 I think I fixed it?
I think my 6 and 8 do this explanation decently 
John said a screenshot lol so I am only going to post one.

Reply:
@dmcnulty27 Regrading 6, I don't think so. Note that in DL all quantified formulas include an implicit universal quantifier, so each quantified formula in DL must pick out “*all individuals* satisfying some complex condition”. 

So "∃r-.C" does not correspond to *some* instance of C that that are related to all instances as you said. Indeed, you are right some instance y of C is r-related to all x. But here we still wanna pick out those xs but not that y. Here is my point.

Also, *something is an instance only if it belongs to a concept*. But here x does not belong to any concept, so you should say "all individuals" instead of "all instances"

Reply:
@dmcnulty27 Regarding 8, I just wanna add some particular application of FMP lol.

Regarding 10, I guess that "a" is an externtial quantifier, so at least one, but not only one lol.